src/js/containers/bulkDownload/dictionary/DataDictionaryContainer.jsx

```diff
@@ -88,7 +88,7 @@ export default class DataDictionaryContainer extends React.Component {
             rows: parsedRows
         }, () => {
             // Default sort
-            this.changeSort('element', 'asc');
+            this.changeSort(this.state.columns[0].raw, 'asc');
```

Comment:
because of related API change, raw column names will be a little different. This seemed to be the only place those names were hard-coded.